### PR TITLE
[FEATURE] Add TypeConverter Examples

### LINKED ISSUE #34: Add TypeConverter Examples
references https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/8

### PR DESCRIPTION
**Pull request by linawolf**
based on https://in2code.de/aktuelles/hidden-objects-an-extbase-actions-uebergeben/

resolves #34
references TYPO3-Documentation/Changelog-To-Doc#8

### CODE REVIEW

File: Classes/Property/TypeConverters/HiddenCommentConverter.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+class HiddenCommentConverter extends PersistentObjectConverter
+{
+    /**
+     * @param mixed $identity
+     * @param string $targetType
+     * @throws TargetNotFoundException
+     * @throws InvalidSourceException
+     * @return Comment
+     */
+    protected function fetchObjectFromPersistence($identity, string $targetType): object
```
Comment (froemken) [inline Classes/Property/TypeConverters/HiddenCommentConverter.php:21]: Use "Comment" as return type instead of "object"

**Review by froemken — CHANGES_REQUESTED** [review]

File: Configuration/Services.yaml [hunk 2]
```diff
@@ -7,3 +7,10 @@ services:
   FriendsOfTYPO3\BlogExample\:
     resource: '../Classes/*'
     exclude: '../Classes/Domain/Model/*'
+
```
Comment (froemken) [inline Configuration/Services.yaml:10]: Remove one empty line

File: Classes/Property/TypeConverters/HiddenCommentConverter.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
```
Comment (froemken) [inline Classes/Property/TypeConverters/HiddenCommentConverter.php:4]: Add Doc header

**Review by froemken — CHANGES_REQUESTED** [review]

File: Configuration/Services.yaml [hunk 4]
```diff
@@ -7,3 +7,10 @@ services:
...
+  FriendsOfTYPO3\BlogExample\Property\TypeConverters\HiddenCommentConverter:
+    tag:
```
Comment (linawolf) [inline Configuration/Services.yaml:12]:
```suggestion
    exclude: '../Classes/Domain/Model/*'
  FriendsOfTYPO3\BlogExample\Property\TypeConverters\HiddenCommentConverter:
    tag:
```

**Pull request merged**